src/styles/blocks/divider.scss

```diff
@@ -0,0 +1,16 @@
+.divider {
+  position: sticky;
+  z-index: -3;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 200px;
+  background-image: url("../img/divider.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+
+  @include onTablet {
+    height: 400px;
+  }
+}
```

Comment:
Do you actually use this class?

Reply:
Yeah. It is an image between main content and footer.